src/test/java/com/edmunds/rest/databricks/service/ClusterServiceTest.java

```diff
@@ -158,4 +160,12 @@ public void edit_whenCalled_changesNodeType() throws IOException, DatabricksRest
     assertEquals(clusterInfo.getNodeTypeId(), "r3.2xlarge");
     assertEquals(clusterInfo.getNumWorkers(), 1);
   }
+
+  @Test
+  public void listEvents_whenCalled_showsEvents() throws IOException, DatabricksRestException {
+    ClusterInfoDTO[] clusterInfoDTO = service.list();
+    String clusterId = clusterInfoDTO[0].getClusterId();
+    ClusterEventsDTO events = service.listEvents(clusterId, new ClusterEventTypeDTO[0], 0, 50);
+    assert(events.getEvents().size() > 0);
```

Comment:
Prefer `assertTrue` over `assert` as the latter only executes if JVM assertions are enabled.